src/components/Layout/Lines.tsx

```diff
@@ -146,6 +153,55 @@ NullishLine.propTypes = {
   depth: PropTypes.number
 }
 
+/**
+ * FilecoinLine
+ */
+
+export const FilecoinLine = ({ label, depth, value }: FilecoinLineProps) => (
+  <Line label={label} depth={depth}>
+    {makeFriendlyBalance(value, 6)} FIL
+  </Line>
+)
+
+type FilecoinLineProps = {
+  value: FilecoinNumber
+} & LineProps
+
+FilecoinLine.propTypes = {
+  value: FILECOIN_NUMBER_PROPTYPE.isRequired,
+  ...LinePropTypes
+}
+
+/**
+ * MethodLine
+ */
+
+export const MethodLine = ({
+  label,
+  depth,
+  actorName,
+  methodNum
+}: MethodLineProps) => (
+  <Line label={label} depth={depth}>
+    <Badge
```

Comment:
Think the Badge is unnecessary here as mentioned above - component still might be nice to have